examples/parlai_chat_task_demo/hydra_configs/conf/base.yaml

```diff
@@ -8,3 +8,6 @@ mephisto:
     world_file: ${task_dir}/demo_worlds.py
     task_description_file: ${task_dir}/task_description.html
     num_conversations: 1
+  task:
+    # We expect to be able to handle 25 concurrent conversations without issue
```

Comment:
is this a typo? Says 25 in the comment but 50 for the value...

Reply:
25 conversations, but they require 2 connected people per. I see the comment is misleading here, can make explicit